src/content/1.7/development/architecture/migration-guide/forms/CQRS-usage-in-forms.md

```diff
@@ -0,0 +1,187 @@
+---
+title: CQRS usage in forms
+weight: 40
+---
+
+# CQRS usage in forms
+
+## Introduction
+
+Assuming that you are already familiar with [CQRS](#) and [CRUD forms]({{< relref "CRUD-form.md" >}}) as this topic only demonstrates the usage of the CQRS approach. To apply it in your forms you need to:
```

Comment:
> [CQRS](#)

Missing link here

Reply:
Removed CQRS links and added todoe's since @sarjon decided to continue on CQRS documentation - I believe he will do that in separate Pull request. After that we will return the CQRS links on this page as I think its quite relevant do use them here.